packages/web/src/components/spa/JobsSection/jobs_section.css

```diff
@@ -1,5 +1,436 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+:root {
+    --main-color: #32baf4;
+    --shadow-color: #467edd;
+    --background-color: #f7f7f7;
+    --font-family: 'Poppins';
+    --text-color: #333;
+    --placeholder-color: white;
+}
+
 .jobs-container {
-    height: 100vh;
+    height: 125vh;
```

Comment:
Instead of `height` use `min-height: 100vh`. This way the page will scale up when job cards are taking more space.